components/breadcrumb/collapse-modes.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Collapse Modes
+page_title: Breadcrumb Collapse Modes
```

Comment:
We seem to be using inconsistent page titles (incuding in different articles in this PR). Let's see which is the right way and stick to it in the future:

* Component Foo Bar
* Component - Foo Bar
* Foo Bar | Component

Reply:
Shall we consider the "Component - Foo Bar" option the best one? I see it is most commonly used, although it looks like most of the overview articles include "Component Foo Bar".

Reply:
Fixed with "Component - Foo Bar"

Reply:
Well, I would vote for "Foo Bar | Component" or "Foo Bar - Component". Not something urgent for now, but worth considering for the future.